src/frontend/src/components/SearchPage/SearchResults.jsx

```diff
@@ -54,7 +57,18 @@ const SearchResults = ({ text, filter }) => {
 
   return (
     <Container className={classes.searchResults}>
-      {data && data.length ? <Timeline pages={[data]} /> : <h1>No results search</h1>}
+      {data && data.length ? (
+        <Timeline
+          pages={data.map((d) => d.values)}
+          nextPage={
+            data[0].values.length * data.length < data[0].results
```

Comment:
This is ugly.  I feel like the logic here to advance pages could probably be moved into the backend vs. doing it here.  Let's just try to get the next page, and ES can return the one it wants.

Reply:
Right, I thought we could add logic to determine whether or not to keep fetching from the front end. But I'll write something in the backend.

Reply:
sorry just to clarify, are you saying to omit the functionality for now? or do you still want me to write a logic in the backend for it